airflow/providers/amazon/aws/operators/sagemaker.py

```diff
@@ -1049,3 +1050,53 @@ def execute(self, context: Context) -> dict | None:
             self.check_interval,
         )
         return best
+
+
+class SageMakerCreateExperimentOperator(SageMakerBaseOperator):
+    """
+    Creates a SageMaker experiment, to be then associated to jobs etc.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:SageMakerCreateExperimentOperator`
+
+    :param name: name of the experiment, must be unique within the AWS account
+    :param description: description of the experiment, optional
+    :param tags: tags to attach to the experiment, optional
+    :param aws_conn_id: The AWS connection ID to use.
+
+    :returns: the ARN of the experiment created, though experiments are referred to by name
+    """
+
+    template_fields: Sequence[str] = (
+        "name",
+        "description",
+        "tags",
+    )
+
+    def __init__(
+        self,
+        *,
+        name: str,
+        description: str | None = None,
+        tags: dict | None = None,
+        aws_conn_id: str = DEFAULT_CONN_ID,
+        **kwargs,
+    ):
+        super().__init__(config={}, aws_conn_id=aws_conn_id, **kwargs)
+        self.name = name
+        self.description = description
+        self.tags = tags or []
```

Comment:
```suggestion
        self.tags = tags or {}
```
Otherwise the calculation for `tag_set` will fail if no `tags` are provided, and this will align with the typing of `tags` too.

Reply:
yeah right my mistake